src/js/messaging/containers/Thread.jsx

```diff
@@ -147,19 +154,16 @@ export class Thread extends React.Component {
     // Exclude the current folder from the list of folders
     // that are passed down to the MoveTo component.
     const moveToFolders = [];
-    this.props.folders.forEach((folder) => {
+    folders.forEach((folder) => {
```

Comment:
You could do this loop in one line if you prefer
`const moveToFolders = _.reject(folders, ['folderId', currentFolder.folderId]);`